tests/cases/elements/toucharea_doubleclick.slint

```diff
@@ -188,9 +194,9 @@ assert_eq(instance.get_touch_double3(), 2);
 slint_testing::mock_elapsed_time(1000);
 instance.set_pointer_event_test("");
 slint_testing::mock_elapsed_time(1000);
-slint_testing::send_mouse_click(&instance, 108., 108.); // first
+slint_testing::send_mouse_click(&instance, 101., 106.); // first
 slint_testing::mock_elapsed_time(20);
-slint_testing::send_mouse_click(&instance, 101., 101.); // second
+slint_testing::send_mouse_click(&instance, 101., 104.); // second
```

Comment:
so that the clicks are really close to eachother.
currently there is a 10px threshold, and the distance before was √((108-101)² + (108-101)²) = 9.89949 which is really close to the threshold.
I want to make sure that this really test that two clicks on different items is not counted as a double click, even with a really small distance  (two pixels now)